nextjs/prompt.js

```diff
@@ -189,6 +189,16 @@ module.exports = {
             ],
             default: "none",
         },
+        {
+            name: "form validation",
```

Comment:
name'in değeri değişken olarak kullanılabiliyordu galiba, o yüzden boşluksuz olması daha iyi olur sanki

Reply:
```suggestion
            name: "form_validation",
```